### PR TITLE
Send Monthly email every month

### PR DESCRIPTION
**Pull request by SSalekin**

### CODE REVIEW

**Comment by sujoydatta** [comment]
ok

**Comment by MuhammadTamzid** [comment]
ok

File: Gemfile [hunk 1]
```diff
@@ -30,6 +30,7 @@ gem "jbuilder",                "2.2.3"
 gem "sdoc",                    "0.4.0", group: :doc
 gem "seed_dump"
 gem "awesome_print", require:"ap"
+gem 'whenever', :require => false
```
Comment (dieunb) [inline Gemfile]: double quote and convention `require: false`

File: app/mailers/user_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class UserMailer < ApplicationMailer
+
+  # Subject can be set in your I18n file at config/locales/en.yml
```
Comment (dieunb) [inline app/mailers/user_mailer.rb]: remove all remark

**Pull request merged**